lms/djangoapps/teams/views.py

```diff
@@ -16,7 +16,6 @@
 from django.utils.functional import cached_property
 from django.utils.translation import ugettext as _
 from django.utils.translation import ugettext_noop
-from django.views import View
```

Comment:
Suspect. Why are we dropping this here? Was it just a wasted import before? (assuming this is probably a standalone change that we dropped stuff in the rest of the PR not just this commit)

Reply:
Yeah, this is just me fixing random pylint stuff. I believe this one was a unused-import